### ISSUE TITLE
HTML encoded text in posts being removed/rendered incorrectly

### ISSUE THREAD

**Issue by humphd** [Seneca-CDOT/telescope#2131]
This might be a duplicate of #1801, and it's interesting that both @yuanLeeMidori and @chrispinkney are both on dev.to.  On [the left](https://dev.to/yuanleemidori/first-attempt-at-intersection-observer-129i) is Yuan's original post, on the right is Telescope:

<img width="1633" alt="Screen Shot 2021-04-12 at 10 05 39 AM" src="https://user-images.githubusercontent.com/427398/114408014-014a6b80-9b77-11eb-9971-1545048af1f5.png">

**Comment by humphd** [comment]
I checked and dev.to is producing valid HTML here, for example:

```html
I move <code>&lt;navbar&gt;</code> into <code>&lt;main&gt;</code>
```

I am starting to think that this might be due to the default `decodeEntities=true` in the [HTML parser](https://github.com/fb55/htmlparser2/wiki/Parser-options#option-decodeentities).  The docs say it's not possible to catch all security issues if you disable this.  So I guess we have to start by seeing if setting `decodeEntities=false` fixes it.  If it does, then we need to decide if we should go that route or not.

**Comment by humphd** [comment]
See also #2322, #2313

**Comment by humphd** [comment]
Fixed by #2337
